### DISCUSSION TITLE
Sagittarius A* Expansion

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#8365] [Ideas]
I feel as if the supermassive black hole in Sagittarius A* has no other purpose than to be in one remnant mission, so I propose that some VERY advanced races (such as the Drak) could use the black hole as a way of transportation (or as an alien race's home). Anyhow, this could have a lot of story impact as a possible explanation for a lot of things like Jump Drives and the Drak Antimatter Cannon. Additionally, this could be a possible wormhole for those advanced enough- the Pug can make wormholes, they could have temporarily "usurped" a black hole to get to the Milky Way, or maybe took a sample of some of the black hole's "matter" for their wormhole technology.
This is just an idea, and not fully fleshed out.

**Comment by Zitchas** [comment]
Good observation. Yes, there's a lot of story potential here, and those are some good ideas. Rest assured that I wouldn't drop a bombshell like "the central black hole of our galaxy is behaving weirdly and defying all our known science about black holes" without having plans for where the story is going from there. More content surrounding Sag A* will be forthcoming later in the Remnant storyline, and will show up in more than a few places....

**Comment by MasterOfGrey** [comment]
If the Xapleaux (see #6439) get in, it will be a prominent part of their superstition/mythology. 
There are no plans to *use* it beyond being a narrative limiter though.
